tools/png2h.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+import sys
+from PIL import Image
+from pathlib import Path
+
+
+def encode_cursor(image_path):
+    img = Image.open(image_path).convert("RGBA")
+    width, height = img.size
+    pixels = img.load()
+
+    num_pixels = width * height
+    num_bytes = (num_pixels + 7) // 8
+
+    data = bytearray(num_bytes)
+    mask = bytearray(num_bytes)
+
+    for y in range(height):
+        for x in range(width):
+            i = y * width + x
+            byte_index = i // 8
+            bit_offset = 7 - (i % 8)
+
+            r, g, b, a = pixels[x, y]
+
+            if a >= 128:
+                mask[byte_index] |= 1 << bit_offset  # opaque
+                lum = int(0.299 * r + 0.587 * g + 0.114 * b)
+                if lum < 128:
+                    data[byte_index] |= 1 << bit_offset  # black pixel
+
+    return data, mask, width, height
+
+
+def to_c_array(name, data):
+    lines = []
+    line = ""
+    for i, b in enumerate(data):
+        line += f"0x{b:02X}, "
+        if (i + 1) % 12 == 0:
+            lines.append(line)
+            line = ""
+    if line:
+        lines.append(line)
+    array_str = "\n    ".join(lines)
+    return f"static const unsigned char {name}[] = {{\n    {array_str}\n}};\n"
+
+
+def main():
+    if len(sys.argv) == 1:
+        print(f"Usage: {sys.argv[0]} [...input.png]")
+        sys.exit(1)
+
+    input_files = sys.argv[1:]
```

Comment:
Use [`argparse`](https://docs.python.org/3/library/argparse.html)

```suggestion
    parser = argparse.ArgumentParser(allow_abbrev=False)
    parser.add_argument("inputs", nargs="+", help="PNG images")
    args = parser.parse_args();

    input_files = args.inputs
```